components/script/fetch.rs

```diff
@@ -121,6 +123,26 @@ impl FetchCanceller {
     }
```

Comment:
In this file I think we should remove the `Trusted<GlobalScope>` from the `DeferredFetchRecord`, because `Trusted` should not be used anywhere on dom_struct or things it contains, and instead pass the global as an argument where it is necessary. Can be done in a follow-up via filing an issue first.

Reply:
Done, removed that from `DeferredFetchRecord`. Note that nearly all of our fetch listeners still use `Trusted<GlobalScope>` which is why it is still present in `FetchLaterListener`